test/test_svd.cc

```diff
@@ -83,15 +83,22 @@ void test_svd_work( Params& params, bool run )
     std::vector<scalar_t> A_data;
     std::vector<scalar_t> Acpy_data;
 
+    // U  is either m-by-min( m, n ) for some vec, or m-by-m for all vec);
+    // VT is either min( m, n )-by-n for some vec, or n-by-n for all vec).
+    int64_t Um  = m;
+    int64_t Un  = jobu == slate::Job::AllVec ? m : min_mn;
+    int64_t VTm = jobu == slate::Job::AllVec ? n : min_mn;
```

Comment:
Shouldn't `VTm` be computed with `jobvt` not `jobu`?

Reply:
Yes. See, this is why we review code.

Reply:
Fixed.